icefall/dataset/librispeech.py

```diff
@@ -0,0 +1,68 @@
+import argparse
```

Comment:
Regarding my previous comment: this file naturally gets merged together with `asr_datamodule` when it’s a local copy

Reply:
I wonder if it might be possible to have Python scripts like this (librispeech.py) localized to the individual recipes somehow, so users feel more free to copy and modify them?

Reply:
Yes, see asr_datamodule.py in gigaspeech recipe after my latest changes, it’s exactly what I did there.